Gemfile

```diff
@@ -27,3 +27,4 @@ gem "rerun"
 
 # Development proxy debugging
 gem "byebug"
+gem "pry"
```

Comment:
Can we add `require: false`?  This should help slim down load times.

Or, add it to a development group.